lib/utility/imsdb_parser.rb

```diff
@@ -44,6 +44,8 @@ def imsdb_url(path)
     end
 
     def populate_script(movie)
+      url_components = movie.url.split('.')
+      return if url_components.include?('pdf')
```

Comment:
Please insert a new line after the guard clause here. As well, we probably want to alter this behaviour to be a little more robust.

```
BLACKLISTED_EXTENSIONS = ['pdf']

return if BLACKLISTED_EXTENSIONS.any? {|extension| url_components.include?(extension) }
```